bcipy/helpers/system_utils.py

```diff
@@ -139,7 +140,8 @@ def configure_logger(
     root_logger = logging.getLogger()
     root_logger.setLevel(log_level)
     handler = logging.FileHandler(logfile, 'w', encoding='utf-8')
-    handler.setFormatter(logging.Formatter('(%(threadName)-9s) %(message)s'))
+    handler.setFormatter(logging.Formatter(
+        '%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
```

Comment:
I like the addition of level name in the logs. I think we may want to retain the threadName, though. I have definitely found it useful in the past what messages were coming from different acquisition threads.